src/views/components/input/index.njk

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
I've not yet included any error states (same for textarea). I figured this would come under a wider piece on implementing the error states (and potentially any considerations we need for the pattern)